test/specs/rules/require-capitalized-comments.js

```diff
@@ -8,6 +10,10 @@ describe('rules/require-capitalized-comments', function() {
         assert(checker.checkString(str).isEmpty());
     }
 
+    function assertOne(str) {
```

Comment:
Yeah it is kind of annoying writing out all that for each test - maybe `assertOne/assertEmpty` can be a helper like `reportAndFix`? Then we change all the rules to that